common/static/common/js/buttonDebounce.js

```diff
@@ -0,0 +1,31 @@
+const tapDebounce = (event) => {
+    let DEBOUNCE_TIMEOUT_IN_SECONDS = 5;
+    let target = event.target;
+    if (target.tapDebounceFormSubmitTimer) {
+      event.preventDefault();
+      return false;
+    }
+
+    let targetDefaultCursor = target.style.cursor;
+
+    target.tapDebounceFormSubmitTimer = setTimeout(
+      function () {
+        target.tapDebounceFormSubmitTimer = null;
+        target.style.cursor = targetDefaultCursor;
+      }.bind(target),
+      DEBOUNCE_TIMEOUT_IN_SECONDS * 1000
+    );
+    target.style.cursor = "wait";
+};
+
+
+const initTapDebounce = () => {
+  document.querySelectorAll("[data-prevent-double-click").forEach(
```

Comment:
Oh, just noticed that there's a closing brace missing in the selector string!
```suggestion
  document.querySelectorAll("[data-prevent-double-click]").forEach(
```